lib/Test/Nginx/Util.pm

```diff
@@ -41,6 +42,10 @@ our $InSubprocess;
 our $RepeatEach = 1;
 our $MAX_PROCESSES = 10;
 
+our $ArchivePath = $ENV{TEST_NGINX_ARCHIVE_PATH};
+our $CurrentTestName = undef;
```

Comment:
I'm not feeling comfortable with this global variable. I think you should just pass the block object along your Perl subroutines and read the test name via `$block->name`.